bin/mail/2016-03-21-teacher-recruitment/generate-inactive-teachers

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+require 'set'
+
+# Inactive Code Studio teacher accounts (not teaching a course, or haven't added students)	US only
+
+US_TEACHERS = Set.new
+SOLR_PARAMS = {
+  q: 'kind_s:user && role_s:teacher',
+  fq: 'create_ip_country_s:"United States"',
+  fl: 'email_s',
+  rows: 10000
+}
+SOLR.query(SOLR_PARAMS).each do |record|
+  US_TEACHERS << record['email_s']
+end
+puts "#{US_TEACHERS.count} US teachers loaded from solr"
+
+query = <<eos
+  SELECT DISTINCT users.email, users.name FROM users
+    LEFT JOIN sections ON (sections.user_id = users.id)
+    LEFT JOIN followers ON (followers.section_id = sections.id)
+    WHERE users.user_type = 'teacher'
+      AND (sections.id IS NULL OR followers.id IS NULL)
```

Comment:
Do you want to include teachers that have created a section, assigned students, and none of the students have ever logged in?

Consider adding a comment explaining what users this query is returning.